build-aux/m4/bitcoin_qt.m4

```diff
@@ -426,14 +427,25 @@ AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
         PKG_CHECK_MODULES([QT_DBUS], [${QT_LIB_PREFIX}DBus], [QT_DBUS_INCLUDES="$QT_DBUS_CFLAGS"; have_qt_dbus=yes], [have_qt_dbus=no])
       fi
     ])
+    BITCOIN_QT_CHECK([
+      AC_CACHE_CHECK(for Qt >= 5.5.1,bitcoin_cv_qt551,
+        PKG_CHECK_EXISTS(Qt5Core < 5.5.1,[bitcoin_cv_qt551=no],[bitcoin_cv_qt551=yes])
```

Comment:
https://cgit.freedesktop.org/pkg-config/tree/pkg.m4.in#n81
```
dnl PKG_CHECK_EXISTS(MODULES, [ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND])`
dnl -------------------------------------------------------------------
dnl Since: 0.18
dnl
dnl Check to see whether a particular set of modules exists. Similar to
dnl PKG_CHECK_MODULES(), but does not set variables or print errors
```

Does this usage of `PKG_CHECK_EXISTS` macro is correct?

Reply:
nm, a version specifier is ok.